src/HtmlElement.php

```diff
@@ -122,7 +122,7 @@ public function __toString()
             $result .= ' ' . $key . '="' . $value . '"';
         }
 
-        if ($this->contents) {
+        if (!is_null($this->contents) and $this->contents != '') {
```

Comment:
Thanks Graham.

@colinodell with the suggested change made, tests are reporting the following: 

```
1) League\CommonMark\Tests\HtmlElementTest::testToString
Failed asserting that two strings are equal.
--- Expected
+++ Actual
@@ @@
-'<p></p><img />'
+'<p></p><img></img>'
```